spec/usage/components/picture_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../../support/utils'
+include Utils
+
+describe 'Picture component', type: :feature, js: true do
+  it 'Renders a picture tag on a page' do
+    class ExamplePage < Matestack::Ui::Page
+      def response
+        components {
+          # Just the tag
+          picture
+
+          # With attributes
+          picture id: 'my-id', class: 'my-class'
+
+          # With a sub-component
+          picture do
+            img path: 'matestack-logo.png'
+          end
+        }
+      end
+    end
+
+    visit '/example'
+
+    static_output = page.html
+
+    expected_static_output = <<~HTML
+      <picture></picture>
+      <picture id="my-id" class="my-class"></picture>
+    HTML
+
+    expect(stripped(static_output)).to include(stripped(expected_static_output))
+    expect(page).to have_xpath("//img[contains(@src,'matestack-logo')]")
```

Comment:
Does this part work? It's defining and expecting an `img`-xpath but there's none in the static output?

Reply:
It should work. The `expect` on line 33 checks the tag defined on line 16-18. 
I couldn't add the expected tag to the `expected_static_output` variable because the img component generates a different path for the logo on each run.

Reply:
Got it, thanks a lot for quick clarification mate!